examples/simple-node/main.tf

```diff
@@ -1,31 +1,31 @@
 module "multiversx-observer" {
-  
+
   #-------source repo
   source  = "ritch2022/multiversx-lighthouse/tencentcloud" #terraform module published in the registry
-  version = "0.3.0" #version of the terraform  module in the registry
-  
+  version = "0.3.0"                                        #version of the terraform  module in the registry
```

Comment:
replace with 0.3.2 as this will be the latest after release push

Reply:
how about removing version setting here? Terraform uses the latest version of module by default.
This will avoid updating version every time we release a new version

Reply:
agreed